Automattic/wp-calypso
# Measure logged-out users opting out of Tracks

We want to add the new event `a8c_cookie_banner_view` and make sure event `a8c_cookie_banner_ok` is firing when the cookie banner is displayed.

With these first changes we add the tracks event `a8c_cookie_banner_view`
<img width="1725" alt="Screenshot 2023-07-20 at 18 43 00" src="https://github.com/Automattic/wp-calypso/assets/44065032/9a5e6bf1-d330-49fa-8b74-8fe7d52f1bb6">



-----
-----

We also add the missing event `a8c_cookie_banner_ok` 

<img width="1724" alt="Screenshot 2023-07-21 at 14 29 29" src="https://github.com/Automattic/wp-calypso/assets/44065032/7bc4dc7e-387a-482d-bafa-89dd30fd9ab2">

<img width="1723" alt="Screenshot 2023-07-21 at 14 29 01" src="https://github.com/Automattic/wp-calypso/assets/44065032/93468862-6987-4006-811e-7458c6e0d5b0">



-----




Related to #

## Proposed Changes

*

## Testing Instructions

Entry point for calypso:

[http://calypso.localhost:3000/start](http://calypso.localhost:3000/start)

We **do** see the cookie banner when using a proxy in Germany.

We then check for the event triggered `a8c_cookie_banner_view`


## Pre-merge Checklist
